frontend/src/app.css

```diff
@@ -32,3 +32,24 @@
 		@apply px-4 py-2 leading-5 text-white transition-colors duration-200 transform bg-gray-700 rounded text-center hover:bg-gray-600 focus:outline-none disabled:cursor-not-allowed disabled:opacity-50;
 	}
 }
+
+#why-classquiz::-webkit-scrollbar {
+	height: 0.8rem;
+	margin-bottom: 5rem;
+}
+
+#why-classquiz::-webkit-scrollbar-track {
+	box-shadow: inset 0 0 10px 10px transparent;
+	border: solid 3px transparent;
+}
+
+#why-classquiz::-webkit-scrollbar-thumb {
+	box-shadow: inset 0 0 10px 10px #374151;
+	border: solid 3px transparent;
+	border-radius: 15px;
+}
+
+#why-classquiz::-webkit-scrollbar-thumb:hover {
+	box-shadow: inset 0 0 10px 10px #555;
+	border: solid 3px transparent;
+}
```

Comment:
Please move the CSS from the `app.css`-file into the `frontend/src/routes/+page.svelte` into a `<style>`-block.